### ISSUE TITLE
Fatal Python error: Illegal instruction

### ISSUE THREAD

**Issue by moskovskiy82** [blakeblackshear/frigate#695]
Tried the stable and beta add ons.
Hass OS running on top of ubuntu 20.04 (x86)
Config is just plain empty default

detectors:
  cpu1:
    type: cpu
  cpu2:
    type: cpu
mqtt:
  host: core-mosquitto.local.hass.io
ffmpeg: {}
cameras: {}

* Starting nginx nginx
   ...done.
Fatal Python error: Illegal instruction
Current thread 0x00007f1f698a8740 (most recent call first):
  File "<frozen importlib._bootstrap>", line 219 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 1101 in create_module
  File "<frozen importlib._bootstrap>", line 556 in module_from_spec
  File "<frozen importlib._bootstrap>", line 657 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 975 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 991 in _find_and_load
  File "<frozen importlib._bootstrap>", line 219 in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1042 in _handle_fromlist
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 36 in <module>
  File "<frozen importlib._bootstrap>", line 219 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 783 in exec_module
  File "<frozen importlib._bootstrap>", line 671 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 975 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 991 in _find_and_load
  File "/opt/frigate/frigate/edgetpu.py", line 15 in <module>
  File "<frozen importlib._bootstrap>", line 219 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 783 in exec_module
  File "<frozen importlib._bootstrap>", line 671 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 975 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 991 in _find_and_load
  File "/opt/frigate/frigate/app.py", line 17 in <module>
  File "<frozen importlib._bootstrap>", line 219 in _call_with_frames_removed
  File "<frozen importlib._bootstrap_external>", line 783 in exec_module
  File "<frozen importlib._bootstrap>", line 671 in _load_unlocked
  File "<frozen importlib._bootstrap>", line 975 in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 991 in _find_and_load
  File "/opt/frigate/frigate/__main__.py", line 7 in <module>
  File "/usr/lib/python3.8/runpy.py", line 87 in _run_code
  File "/usr/lib/python3.8/runpy.py", line 194 in _run_module_as_main

**Comment by blakeblackshear** [comment]
It looks like tflite may not support you hardware. What are you running on?

**Comment by moskovskiy82** [comment]
> It looks like tflite may not support you hardware. What are you running on?

An old C2D E6750 if i recall correctly

**Comment by moskovskiy82** [comment]
@blakeblackshear, this solution works #49

**Comment by parhedberg** [comment]
I think that i am having the same issue.. But i can't get the solution above to work. 

It tries to install the "old pc" version, but it takes forever, and then fails. No information what i can find in logs.. 

Trying to set this up with a https://www.dell.com/se/offentlig/p/optiplex-160/pd
Did not have any trouble running Frigate before the breaking changes update a while ago..


Only error that can be related is in host log: 

`[ 8906.817380] traps: python3[3347] trap invalid opcode ip:7fa68706e10c sp:7ffc01c5e720 error:0 in _pywrap_tensorflow_interpreter_wrapper.cpython-38-x86_64-linux-gnu.so[7fa687056000+2c9000]`

Any idéas? I get my Coral USB dongle tomorrow... Would love for it to work then.. :)

**Comment by parhedberg** [comment]
After 10 or so restarts the plugin is installed.. But There is only a green picture. 


This is my config;

`detectors:
  cpu1:
    type: cpu
  cpu2:
    type: cpu
mqtt:
  host: 172.30.32.1
  user: parhedberg
  password: br3db4nD
ffmpeg:
  global_args: '-hide_banner -loglevel warning'
  hwaccel_args:
    - '-hwaccel'
    - vaapi
    - '-hwaccel_device'
    - /dev/dri/renderD128
    - '-hwaccel_output_format'
    - yuv420p
  input_args:
    - '-avoid_negative_ts'
    - make_zero
    - '-fflags'
    - nobuffer
    - '-flags'
    - low_delay
    - '-strict'
    - experimental
    - '-fflags'
    - +genpts+discardcorrupt
    - '-use_wallclock_as_timestamps'
    - '1'
  output_args:
    detect: '-f rawvideo -pix_fmt yuv420p'
    record: >-
      -f segment -segment_time 60 -segment_format mp4 -reset_timestamps 1
      -strftime 1 -c copy -an
    clips: >-
      -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1
      -strftime 1 -c copy -an
    rtmp: '-c copy -f flv'
cameras:
  framsidan:
    ffmpeg:
      inputs:
        - path: 'rtsp://admin:!H0usT0N@192.168.0.15:88/videoSub'
          roles:
            - detect
            - rtmp
    width: 1280
    height: 720
    fps: 5
    clips:
      enabled: true
      pre_capture: 5
      post_capture: 5
      objects:
        - person
      retain:
        default: 10
        objects:
          person: 15
    record:
      enabled: true
      retain_days: 5
`

**Comment by parhedberg** [comment]
I got it working after some time. Struggeling with settings for hwaccel and first to get the "old x86_64" version even installed.. But now it works fine!

**Comment by moskovskiy82** [comment]
> I got it working after some time. Struggeling with settings for hwaccel and first to get the "old x86_64" version even installed.. But now it works fine!

pdecat was working hard to get it up and running. Mine is working flawlessly now. Hope @blakeblackshear will integrate the solution into the main repo so we will get updates

**Comment by parhedberg** [comment]
I have now received my Coral device, and tries to use it on the same setup that needed "old cpu" version. 

Then i get a different error:

```
 * Starting nginx nginx
   ...done.
frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
peewee_migrate                 INFO    : There is nothing to migrate
frigate.app                    INFO    : Camera processor started for framsidan: 41
detector.coral                 INFO    : Starting detection process: 38
frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate.app                    INFO    : Capture process started for framsidan: 42
Fatal Python error: Illegal instruction
Thread 0x00007f88488c4700 (most recent call first):
  File "/usr/lib/python3.8/threading.py", line 302 in wait
  File "/usr/lib/python3.8/multiprocessing/queues.py", line 227 in _feed
  File "/usr/lib/python3.8/threading.py", line 870 in run
  File "/usr/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/usr/lib/python3.8/threading.py", line 890 in _bootstrap
Current thread 0x00007f885cffd740 (most recent call first):
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 108 in __init__
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 152 in load_delegate
  File "/opt/frigate/frigate/edgetpu.py", line 63 in __init__
  File "/opt/frigate/frigate/edgetpu.py", line 124 in run_detector
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108 in run
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315 in _bootstrap
  File "/usr/lib/python3.8/multiprocessing/popen_fork.py", line 75 in _launch
  File "/usr/lib/python3.8/multiprocessing/popen_fork.py", line 19 in __init__
  File "/usr/lib/python3.8/multiprocessing/context.py", line 277 in _Popen
  File "/usr/lib/python3.8/multiprocessing/context.py", line 224 in _Popen
  File "/usr/lib/python3.8/multiprocessing/process.py", line 121 in start
  File "/opt/frigate/frigate/edgetpu.py", line 186 in start_or_restart
  File "/opt/frigate/frigate/edgetpu.py", line 169 in __init__
  File "/opt/frigate/frigate/app.py", line 156 in start_detectors
  File "/opt/frigate/frigate/app.py", line 223 in start
  File "/opt/frigate/frigate/__main__.py", line 15 in <module>
  File "/usr/lib/python3.8/runpy.py", line 87 in _run_code
  File "/usr/lib/python3.8/runpy.py", line 194 in _run_module_as_main
frigate.mqtt                   INFO    : MQTT connected
frigate.mqtt                   INFO    : Turning on snapshots for framsidan via mqtt
frigate.watchdog               INFO    : Detection appears to have stopped. Restarting frigate
frigate.app                    INFO    : Stopping...
frigate.events                 INFO    : Exiting event processor...
frigate.record                 INFO    : Exiting recording maintenance...
frigate.object_processing      INFO    : Exiting object processor...
frigate.events                 INFO    : Exiting event cleanup...
frigate.watchdog               INFO    : Exiting watchdog...
```

Only thing i changed was:

Old config:
```
detectors:
  cpu1:
    type: cpu
  cpu2:
    type: cpu
```

New config:
```
detectors:
  coral:
    type: edgetpu
    device: usb
```


The system can find the USB-dongle, and it is listed in lsusb command, so it looks ok so far..

Is this a problem with the legacy "old cpu" version?

**Comment by blakeblackshear** [comment]
You are still using the old cpu version of the container right? That looks like the same error that requires the old cpu version. You still need the old cpu version even with a coral.

**Comment by parhedberg** [comment]
Yes, i am still using the old cpu version..

Did a test with the regular version, but that version fails a bit earlier in the process... Right after ".. done" at the beginning..

This failes later, but with almost the same error.

**Comment by solarpowertoast** [comment]
I'm having a similar issue.  Running on a Xeon from 2010 that also doesn't support AVX.  Using the Home Assistant addon of Frigate for old CPUs.  I also have a Coral TPU plugged in.  Here is my log.  It seems like the same issue as @parhedberg 

` * Starting nginx nginx`
`   ...done.`
`frigate.app                    INFO    : Creating directory: /tmp/cache`
`frigate.app                    WARNING : Camera porch has rtmp enabled, but rtmp is not assigned to an input.`
`Starting migrations`
`peewee_migrate                 INFO    : Starting migrations`
`There is nothing to migrate`
`peewee_migrate                 INFO    : There is nothing to migrate`
`frigate.app                    INFO    : Camera processor started for porch: 42`
`frigate.app                    INFO    : Capture process started for porch: 44`
`frigate.mqtt                   INFO    : MQTT connected`
`Fatal Python error: Illegal instruction`
`Thread 0x00007f0447832700 (most recent call first):`
`  File "/usr/lib/python3.8/multiprocessing/queues.py", line 224 in _feed`
`  File "/usr/lib/python3.8/threading.py", line 870 in run`
`  File "/usr/lib/python3.8/threading.py", line 932 in _bootstrap_inner`
`  File "/usr/lib/python3.8/threading.py", line 890 in _bootstrap`
`Current thread 0x00007f045bf6b740 (most recent call first):`
`  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 108 in __init__`
`  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 152 in load_delegate`
`  File "/opt/frigate/frigate/edgetpu.py", line 63 in __init__`
`  File "/opt/frigate/frigate/edgetpu.py", line 124 in run_detector`
`  File "/usr/lib/python3.8/multiprocessing/process.py", line 108 in run`
`  File "/usr/lib/python3.8/multiprocessing/process.py", line 315 in _bootstrap`
`  File "/usr/lib/python3.8/multiprocessing/popen_fork.py", line 75 in _launch`
`  File "/usr/lib/python3.8/multiprocessing/popen_fork.py", line 19 in __init__`
`  File "/usr/lib/python3.8/multiprocessing/context.py", line 277 in _Popen`
`  File "/usr/lib/python3.8/multiprocessing/context.py", line 224 in _Popen`
`  File "/usr/lib/python3.8/multiprocessing/process.py", line 121 in start`
`  File "/opt/frigate/frigate/edgetpu.py", line 186 in start_or_restart`
`  File "/opt/frigate/frigate/edgetpu.py", line 169 in __init__`
`  File "/opt/frigate/frigate/app.py", line 156 in start_detectors`
`  File "/opt/frigate/frigate/app.py", line 223 in start`
`  File "/opt/frigate/frigate/__main__.py", line 15 in <module>`
`  File "/usr/lib/python3.8/runpy.py", line 87 in _run_code`
`  File "/usr/lib/python3.8/runpy.py", line 194 in _run_module_as_main`
`frigate.watchdog               INFO    : Detection appears to have stopped. Restarting frigate`
`frigate.app                    INFO    : Stopping...`
`frigate.events                 INFO    : Exiting event processor...`
`frigate.record                 INFO    : Exiting recording maintenance...`
`frigate.object_processing      INFO    : Exiting object processor...`
`frigate.watchdog               INFO    : Exiting watchdog...`
`frigate.events                 INFO    : Exiting event cleanup...`
`frigate.app                    INFO    : Stopping...`

**Comment by solarpowertoast** [comment]
If I change the detector from the edgetpu to cpu, I get no errors in the log and it seems to work ok.  However, as soon as I change back to using the USB Coral edgetpu I get the illegal instruction error in the logs and Frigate stops after a few minutes.  Do I need to install drivers for the Coral USB for it to work?  I'm running on Debian 10.8 on a Xeon box from 2010.

**Comment by pdecat** [comment]
There was much activity on the tflite repository recently that may have broken things, and maybe even more recent changes to [fix things](https://github.com/tensorflow/tensorflow/commit/c1813ca4e25fb05bfd3b4a02634dd551a768f431) (about 1 hour ago): https://github.com/tensorflow/tensorflow/commits/master/tensorflow/lite

@parhedberg @solarpowertoast Could you try to rebuild your images?

Maybe I should have pinned a known working revision to check out in the Dockerfile used to locally re-build tflite.
https://github.com/blakeblackshear/frigate-hass-addons/pull/12/files#diff-e87a105d6223f52363790368b7f9010d1f79325f1a87be6cbc1809f48b5725fbR17

**Comment by solarpowertoast** [comment]
I just rebuilt my image as well as uninstalled and reinstalled and no change.  Still getting the same errors.  I'm not as familiar with using dockerfiles but I'll see if I can't figure out how to re-build tflite using that.  Thanks for the quick feedback @pdecat

**Comment by pdecat** [comment]
Ok, looks like I missed an important step, my current proposed `Dockerfile` does not actually rebuild the `libedgetpu.so.1.0` library:

```
docker run -ti frigate-test:latest sh
# find / -name libedgetpu.so.1.0
/usr/lib/x86_64-linux-gnu/libedgetpu.so.1.0
# ls -l /usr/lib/x86_64-linux-gnu/libedgetpu.so.1.0
-rw-r--r-- 1 root root 983536 Nov  2 18:58 /usr/lib/x86_64-linux-gnu/libedgetpu.so.1.0
# dpkg -S /usr/lib/x86_64-linux-gnu/libedgetpu.so.1.0
libedgetpu1-max:amd64: /usr/lib/x86_64-linux-gnu/libedgetpu.so.1.0
```

My [tests from December](https://github.com/blakeblackshear/frigate/issues/49#issuecomment-752720559) were with an actual Coral TPU plugged into a PCIe slot of my old system.

When I started [new tests in January](https://github.com/blakeblackshear/frigate/issues/49#issuecomment-756212050), I did not have the Coral TPU in that same system.

Without my change, it used to crash on [line #14 of edgetpu.py](https://github.com/blakeblackshear/frigate/blob/v0.8.0-beta3/frigate/edgetpu.py#L14):  (tag v0.8.0-beta3 at that time). My change made it not crash during that same  `tflite_runtime.interpreter` import, not sure why now.

But as I did not have a TPU in my system, I never reached [line #63 of edgetpu.py](https://github.com/blakeblackshear/frigate/blob/v0.8.1/frigate/edgetpu.py#L63) where it actually loads the `libedgetpu.so.1.0` library which is causing the crash you are currently seeing.

More work needed then...

**Comment by pdecat** [comment]
I pushed a change to my main branch to actually rebuild libedgetpu.so on the target platform: https://github.com/pdecat/frigate-hass-addons/commit/04a364ff04e2ba824481579ade560e1738cefe11

Will port it to [my PR](https://github.com/blakeblackshear/frigate-hass-addons/pull/12) if you can confirm it works for you.

**Comment by solarpowertoast** [comment]
I tried updating the HA addon to 1.9.6 but it just spun and never finished.  So I uninstalled the addon and when I try to reinstall, I get this error that stops installation:

`Failed to install addon'

'The command '/bin/sh -c cd /libedgetpu && make libedgetpu' returned a non-zero code: 2`

**Comment by pdecat** [comment]
You should be able to see more verbose logs about the compilation failure in the supervisor output with `docker logs hassio_supervisor`.

**Comment by pdecat** [comment]
Alright, I believe I've reproduced the compilation error on my old Intel Core E4400 (it took ~35 min before failing):

```
21-02-25 07:27:14 INFO (SyncWorker_2) [supervisor.docker.interface] Updating image b4616116/amd64-addon-frigate-beta-oldcpu:1.2 to b4616116/amd64-addon-frigate-beta-oldcpu:1.9.5
21-02-25 07:27:14 INFO (SyncWorker_2) [supervisor.docker.addon] Starting build for b4616116/amd64-addon-frigate-beta-oldcpu:1.9.5
21-02-25 08:02:41 ERROR (SyncWorker_2) [supervisor.docker.addon] Can't build b4616116/amd64-addon-frigate-beta-oldcpu:1.9.5: The command '/bin/sh -c cd /libedgetpu && make libedgetpu' returned a non-zero code: 2
21-02-25 08:02:41 ERROR (SyncWorker_2) [supervisor.docker.addon] Build log:
...
 ERROR: /libedgetpu/executable/BUILD:31:22: Generating flatbuffer files for executable_fbs_srcs: //executable:executable_fbs_srcs failed: (Illegal instruction): process-wrapper failed: error executing command
  (cd /root/.cache/bazel/_bazel_root/c416b74bea5dee01fd24eeae7231a105/sandbox/processwrapper-sandbox/44/execroot/libedgetpu && \
  exec env - \
    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
    PYTHON_BIN_PATH=/usr/bin/python3 \
    TMPDIR=/tmp \
  /root/.cache/bazel/_bazel_root/install/1a4a2fac02d50c77031d44c0d91b8920/process-wrapper '--timeout=0' '--kill_delay=15' /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh; for f in executable/executable.fbs; do bazel-out/host/bin/external/flatbuffers/flatc --gen-object-api --force-empty --gen-mutable  -c -o bazel-out/k8-opt/bin/executable $f; done') process-wrapper failed: error executing command
  (cd /root/.cache/bazel/_bazel_root/c416b74bea5dee01fd24eeae7231a105/sandbox/processwrapper-sandbox/44/execroot/libedgetpu && \
  exec env - \
    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
    PYTHON_BIN_PATH=/usr/bin/python3 \
    TMPDIR=/tmp \
  /root/.cache/bazel/_bazel_root/install/1a4a2fac02d50c77031d44c0d91b8920/process-wrapper '--timeout=0' '--kill_delay=15' /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh; for f in executable/executable.fbs; do bazel-out/host/bin/external/flatbuffers/flatc --gen-object-api --force-empty --gen-mutable  -c -o bazel-out/k8-opt/bin/executable $f; done')

/bin/bash: line 1:   411 Illegal instruction     (core dumped) bazel-out/host/bin/external/flatbuffers/flatc --gen-object-api --force-empty --gen-mutable -c -o bazel-out/k8-opt/bin/executable $f

Target //tflite/public:libedgetpu_direct_all.so failed to build
```

**Comment by parhedberg** [comment]
I got the same issue when upgrading.. The build fails..

**Comment by pdecat** [comment]
@moskovskiy82 do *not* upgrade your working addon to anything higher than v1.9.4. I'll resume working on this ASAP.

**Comment by solarpowertoast** [comment]
Tried again to install (version 1.9.6) and this is the error I'm getting now.  Not sure if this helps or means anything for you.

`<html> <head><title>504 Gateway Time-out</title></head> <body> <center><h1>504 Gateway Time-out</h1></center> <hr><center>nginx</center> </body> </html> <!-- a padding to disable MSIE and Chrome friendly error page --> <!-- a padding to disable MSIE and Chrome friendly error page --> <!-- a padding to disable MSIE and Chrome friendly error page --> <!-- a padding to disable MSIE and Chrome friendly error page --> <!-- a padding to disable MSIE and Chrome friendly error page --> <!-- a padding to disable MSIE and Chrome friendly error page -->`

**Comment by pdecat** [comment]
Hi @parhedberg @solarpowertoast, I've pushed version 1.9.7 which should fix the build issues on your platforms (works on my machine :tm: :sweat_smile: )

**Comment by solarpowertoast** [comment]
It compiles and installs successfully.  However it still stops detecting after a few seconds.  I've tried the Coral TPU plugged into USB 3 and USB 2 ports with no difference.  It does show up on the host when I run "lsusb"

`  * Starting nginx nginx
   ...done.
frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
peewee_migrate                 INFO    : There is nothing to migrate
frigate.app                    INFO    : Camera processor started for porch: 40
frigate.app                    INFO    : Camera processor started for driveway: 41
frigate.app                    INFO    : Capture process started for porch: 42
frigate.app                    INFO    : Capture process started for driveway: 43
detector.coral                 INFO    : Starting detection process: 37
frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate.mqtt                   INFO    : MQTT connected
frigate.edgetpu                INFO    : No EdgeTPU detected.
Process detector:coral:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 152, in load_delegate
    delegate = Delegate(library, options)
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 111, in __init__
    raise ValueError(capture.message)
ValueError
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/frigate/frigate/edgetpu.py", line 124, in run_detector
    object_detector = LocalObjectDetector(tf_device=tf_device, num_threads=num_threads)
  File "/opt/frigate/frigate/edgetpu.py", line 63, in __init__
    edge_tpu_delegate = load_delegate('libedgetpu.so.1.0', device_config)
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 154, in load_delegate
    raise ValueError('Failed to load delegate from {}\n{}'.format(
ValueError: Failed to load delegate from libedgetpu.so.1.0
frigate.watchdog               INFO    : Detection appears to have stopped. Restarting frigate
frigate.app                    INFO    : Stopping...
frigate.events                 INFO    : Exiting event processor...
frigate.record                 INFO    : Exiting recording maintenance...
frigate.object_processing      INFO    : Exiting object processor...
frigate.events                 INFO    : Exiting event cleanup...
frigate.watchdog               INFO    : Exiting watchdog...    `

**Comment by solarpowertoast** [comment]
I did test it using CPU as the detector and it does work that way successfully identifying me on my porch.  So I just have some issue with the Coral USB TPU.

**Comment by parhedberg** [comment]
I disabled protection mode, and got it working for me..

**Comment by pdecat** [comment]
Excellent news! :tada:

**Comment by pdecat** [comment]
I've updated [my PR](https://github.com/blakeblackshear/frigate-hass-addons/pull/12) with those changes and brought it in sync with the current stable addon release (based on frigate 0.8.3).

I've also published a new 1.12.1 release from my main branch with the same changes: https://github.com/pdecat/frigate-hass-addons/commit/e77e7ec6b863f9daff6c994f3b510f3ef1073116

**Comment by solarpowertoast** [comment]
Great work @pdecat ! Everything seems to work now! Thanks so much for the hard work you did getting this working!
